### PR TITLE
4011 moble app management page build

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Built vertical page for 'Mobile app management'.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/enterprise
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1YMfnGmUWZqs89uVyREdH9BzkHasp4b3RDV0FzS75K3I/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4011

### CODE REVIEW

File: templates/enterprise/index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <h2>Secure mobile application management</h2>
```
Comment (bethcollins92) [inline templates/enterprise/index.html:89]: The child of a `row` should always be a `col` (you can just do `col-12` or use the `u-fixed-width` class instead)

File: templates/enterprise/index.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+      <div class="col-6">
+        <h4>Mobile apps are central to businesses undergoing digital transformation. However, security and data protection is costly with classical mobile app management solutions. Anbox Cloud enables compliant and low-Capex mobile application management from the cloud.</h4>
+      </div>
+      <div class="col-6">
+        <p>Enterprises around the world are undergoing rapid digital transformation. For enterprises with a mobile workforce, digital transformation drives the increasing use of corporate productivity apps on smartphones and tablets.</p>
+        <p>However, with corporate-provided mobile devices, creating strict boundaries between private and corporate data is a challenge. Employees may use corporate devices to private ends, thereby exposing sensitive data.</p>
+        <p>Enterprise IT leaders need to put a mobile strategy in place that fosters rapid digital transformation while protecting corporate data and employee privacy. Mobile application management (MAM) offers a secure way to distribute, secure, monitor and update mobile apps for the digital workplace.</p>
+        <a>Learn about MAM with Anbox Cloud&nbsp;&rsaquo;</a>
```
Comment (bethcollins92) [inline templates/enterprise/index.html:41]: This link has no `href` were you supposed to remove it?

**Review by bethcollins92 — APPROVED** [review]
LGTM with a couple of comments in the code 👍🏼

**Comment by spencerbygraves** [comment]
Thanks @petesfrench, looks good.

**Pull request merged**